app/models/solidus_paypal_commerce_platform/payment_source.rb

```diff
@@ -5,6 +5,40 @@ class PaymentSource < SolidusSupport.payment_source_parent_class
     self.table_name = "paypal_commerce_platform_sources"
     validates :paypal_order_id, :payment_method_id, presence: true
 
+    def paypal_funding_type_map
+      @paypal_funding_type_map ||= {
+        applepay: 'Apple Pay',
+        bancontact: 'Bancontact',
+        blik:	'BLIK',
+        boleto: 'Boleto',
+        card: 'Credit or debit cards',
+        credit:	'PayPal Credit',
+        eps:	'eps',
+        giropay: 'giropay',
+        ideal: 'iDEAL',
+        itau: 'Itau',
+        maxima: 'Maxima',
+        mercadopago:	'Mercado Pago',
+        mybank:	'MyBank',
+        oxxo: 'Oxxo',
+        p24:	'Przelewy24',
+        paylater:	'Pay Later',
+        paypal: 'PayPal',
+        payu: 'PayU',
+        sepa: 'SEPA-Lastschrift',
+        sofort:	'Sofort',
+        trustly: 'Trustly',
+        venmo: 'Venmo',
+        verkkopankki: 'Verkkopankki',
+        wechatpay: 'WeChat Pay',
+        zimpler: 'Zimpler'
+      }
+    end
```

Comment:
What do you think about moving this into an enum field and the translation to an en.yml locale file?

Reply:
Moving it to an enum field sounds a good idea @DanielePalombo. I am not sure what you mean by the translation question however.

Reply:
`enum` allows specifying only the values and the keys, in order to make them international, we should use the I18n file for the translation.

Reply:
I understand now, but perhaps only `card` and `credit` would need translating as the others are registered names and wouldn't need translating, right?

Reply:
Right!

Reply:
I updated the PR now, what do you think @DanielePalombo?